internal/impl/postgresql/pglogicalstream/pglogrepl.go

```diff
@@ -457,14 +457,15 @@ func CreatePublication(ctx context.Context, conn *pgconn.PgConn, publicationName
 
 // GetPublicationTables returns a list of tables currently in the publication
 // Arguments, in order: list of the tables, exist for all tables, errror
-func GetPublicationTables(ctx context.Context, conn *pgconn.PgConn, publicationName string) ([]string, bool, error) {
+func GetPublicationTables(ctx context.Context, conn *pgconn.PgConn, publicationName, schema string) ([]string, bool, error) {
 	query, err := sanitize.SQLQuery(`
 		SELECT DISTINCT
 			tablename as table_name
 		FROM pg_publication_tables
 		WHERE pubname = $1
+		AND schemaname = $2
```

Comment:
This should _return_ the schema names rather than filtering on them, right?

Reply:
Yes, done.

Reply:
Oh, I see. Thanks for catching that!